mantidproject/mantidimaging
# Monitor Normalisation: Warn if there is no log values loaded

Fixes #647 

To Test:
- Load data without log data
- Open operation
- Select monitor normalization
- There should be a warning in the bottom left for both previews and the normal apply
- Repeat with data that has log data
